vpr/src/place/initial_noc_placment.h

```diff
@@ -3,13 +3,19 @@
 #define VTR_INITIAL_NOC_PLACMENT_H
 
 #include "vpr_types.h"
+#include "vtr_vector_map.h"
```

Comment:
vtr vector map is not directly used in this header file. This header include should be removed.